Update virtual-machines-common-sizes-general.md

axelgMS opened the pull request:
RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's  "Xeon E5-2673 v3" and "Xeon E5-2673 v4"

We’re having customers « complaining » about getting E5-2673 v4 processors with Dv2 series
Doc says that “It is based on the latest generation 2.4 GHz Intel Xeon® E5-2673 v3 (Haswell) processor,”

Can we please make it clear and use wording like the one suggested by Christopher?
We should make it clearer that a given VM series may run on future processors as well, not just for the F-series, but for the other series as well. We make similar statements for the Dv2’s for example.

azurecla commented:
Hi __@paulomarquesc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@paulomarquesc : Thanks for your contribution to the Azure documentation! The author, @MGoedtel, has been notified to review your proposed change.

MGoedtel commented:
@paulomarquesc thank you for your time to review and contribute to this article.  I've reviewed your suggestion and because we've already addressed that other concern in a different PR, this PR is not necessary so I'm going to kindly reject this PR.  @Jim-parker - ping me offline if you require specific details.

Jim-Parker commented:
@paulomarquesc @MGoedtel  Paulo, are you okay with me closing this PR based on Matt's info?

axelgMS committed:
Update virtual-machines-common-sizes-general.md

RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's  "Xeon E5-2673 v3" and "Xeon E5-2673 v4"

We’re having customers « complaining » about getting E5-2673 v4 processors with Dv2 series
Doc says that “It is based on the latest generation 2.4 GHz Intel Xeon® E5-2673 v3 (Haswell) processor,”

Can we please make it clear and use wording like the one suggested by Christopher?
We should make it clearer that a given VM series may run on future processors as well, not just for the F-series, but for the other series as well. We make similar statements for the Dv2’s for example.

axelgMS committed:
RE: Sev 3: Id 54668536: F series VM's having inconsistent CPU's "Xeon E5-2673 v3" and "Xeon E5-2673 v4"

Thanks Axel, appreciate the work.  I just took a look at the PR and to make sure we’re accurate I’d like to suggest some additional changes.  Here’s the updated text in the PR:

Dv3-series, Dv2-series, a follow-on to the original D-series, features a more powerful CPU. The Dv2-series CPU is about 35% faster than the D-series CPU. It is based on the latest generation 2.4 GHz Intel Xeon® E5-2673 v3 (Haswell) or E5-2673 v4 (Broadwell) processors, and with the Intel Turbo Boost Technology 2.0, can go up to 3.1 GHz. The Dv2-series has the same memory and disk configurations as the D-series.

And here’s my suggested revision:

Dv3-series, Dv2-series, a follow-on to the original D-series, features a more powerful CPU. The Dv2-series CPU is about 35% faster than the D-series CPU. It is based on the latest generation 2.4 GHz Intel Xeon® E5-2673 v3 2.4 GHz (Haswell) or E5-2673 v4 2.3 GHz  (Broadwell) processors, and with the Intel Turbo Boost Technology 2.0, can go up to 3.1 GHz. The Dv2-series has the same memory and disk configurations as the D-series.

The Dv3-series features the same processor(s) as the Dv2-series, but in a hyper-threaded configuration, providing a better value proposition for most general purpose workloads, and bringing the Dv3 into alignment with the general purpose VMs of most other clouds.  Memory has been expanded (from ~3.5 GiB/vCPU to 4 GiB/vCPU) while disk and network limits have been adjusted on a per core basis to align with the move to hyperthreading.  The Dv3 no longer has the high memory VM sizes of the D/Dv2 families, those have been moved to the new Ev3 family.
